packages/sc/build.sh

```diff
@@ -1,14 +1,12 @@
-TERMUX_PKG_HOMEPAGE="http://www.ibiblio.org/pub/Linux/apps/financial/spreadsheet/!INDEX.html"
+TERMUX_PKG_HOMEPAGE="https://github.com/n-t-roff/sc"
 TERMUX_PKG_DESCRIPTION="A vi-like spreadsheet calculator"
 TERMUX_PKG_LICENSE="Public Domain"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=7.16
-TERMUX_PKG_REVISION=4
-TERMUX_PKG_SRCURL=http://www.ibiblio.org/pub/Linux/apps/financial/spreadsheet/sc-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=1997a00b6d82d189b65f6fd2a856a34992abc99e50d9ec463bbf1afb750d1765
+TERMUX_PKG_VERSION=7.16_1.1.2
```

Comment:
Please use a more saner version string, in which an underscore (`_`) should be avoided. I propose something like:

```bash
_VERSION=7.16_1.1.2
TERMUX_PKG_VERSION=${_VERSION//_/-}
```